src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
Currently here by using `flex-wrap: wrap` you are wrapping children elements depending on screen's width which is agains the requirement for this task (always display 4 cards in a row). I have a few comments on this:
- the problem with positioning of elements here is more flawed than just this one css rule
- ideally we would use `display: grid` layout as we are trying to create a grid pattern here
- however we cannot as without `gap` property it would be rather difficult
- I would suggest rewriting this whole container and also adding a `div` wrapper for each group of 4 children:
```
.main-wrapper {
  display: flex;
  justify-content: center;
  flex-direction: column;
  padding: 60px 0;
  box-sizing: content-box;
  max-width: 1024px;
  margin: 0 auto;

  &__cards {
    display: flex;
    justify-content: center;
    flex-shrink: 0;

    &:not(:first-child) {
      margin-top: 50px;
    }
  }
}
```